lti_redirect/utils/debugpy.py

```diff
@@ -0,0 +1,18 @@
+from decouple import config
+from distutils.util import strtobool
```

Comment:
It looks like distutils will be removed in Python 3.12. We probably should just remove this and use an alternative pattern now.

Maybe something like this

```
def config_to_bool(value):
    return str(value).lower() in ('true', '1', 'yes', 'on')
```

Reply:
Since `strtobool` is only used in `forcebool`, make the logic of the suggested `config_to_bool` part of `forcebool` directly.